Mathlib/Probability/CondVar.lean

```diff
@@ -58,7 +58,7 @@ lemma condVar_of_sigmaFinite [SigmaFinite (μ.trim hm)] :
       else 0 := condExp_of_sigmaFinite _
 
 lemma condVar_of_stronglyMeasurable [SigmaFinite (μ.trim hm)]
-    (hX : StronglyMeasurable[m] X) (hXint : Integrable ((X - μ[X | m]) ^ 2) μ) :
+    (hX : StronglyMeasurable[m] X) (hXint : Integrable ((X - μ[X|m]) ^ 2) μ) :
```

Comment:
Likewise here; I guess it's not clear to me what the rules are on putting spaces around symbols like this -- is the idea that the vertical bar acts more like punctuation than a mathematical operator?

Reply:
Thanks for looking closely! So, that seems like this should use spaces, right (matching the remainder of the file). So I'll revert this change - and it might be interesting to investigate why the linter suggests to remove them.